orc8r/cloud/go/swagger/combine_test.go

```diff
@@ -0,0 +1,121 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package swagger_test
+
+import (
+	"flag"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"magma/orc8r/cloud/go/swagger"
+	"magma/orc8r/cloud/go/tools/combine_swagger/spec"
+	"magma/orc8r/cloud/go/tools/swaggergen/generate"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/yaml.v2"
+)
+
+func init() {
+	_ = flag.Set("alsologtostderr", "true") // uncomment to view logs during test
```

Comment:
Two things

1. Agreed this could be `logtostderr` instead of `alsologtostderr` [0], but since we use the latter throughout the codebase, let's leave it as-is for now and we can make a single change-over if desired later in a single PR
2. @AndyLKhuu this line should be commented-out when it's merged into master (so you'll need to comment it out now)

[0] Andy if you're interested: https://godoc.org/github.com/golang/glog